fastlane/Fastfile

```diff
@@ -72,36 +79,20 @@ platform :android do
     )
   end
 
-  desc "Make github release"
+  desc "Creates github release and updates the latest tag"
   lane :github_release do |options|
-    release_version = options[:version]
-    fail ArgumentError, "missing version" unless release_version
-
-    begin
-      changelog = File.read("../CHANGELOG.latest.md")
-    rescue
-      fail "please add a CHANGELOG.latest.md file before calling this lane"
-    end
-
-    is_prerelease = release_version.include?("-")
-
-    set_github_release(
-      repository_name: "revenuecat/purchases-android",
-      api_token: ENV["GITHUB_TOKEN"],
-      name: release_version,
-      tag_name: "#{release_version}",
-      description: changelog,
-      commitish: "main",
-      upload_assets: [],
-      is_prerelease: is_prerelease
+    create_github_release(
+      version: options[:version],
+      repo_name: repo_name,
+      github_api_token: ENV["GITHUB_TOKEN"],
+      changelog_latest_path: changelog_latest_path
     )
-
     tag_release_with_latest_if_needed(release_version: release_version)
   end
 
   desc "Upload and close a release"
   lane :deploy do |options|
-    version = android_get_version_name(gradle_file: gradle_file_path)
+    version = current_version_number
```

Comment:
I changed to use a `.version` instead of getting the version directly from gradle. I think this is good to make all our SDKs consistent since this is platform agnostic but do lmk if you think otherwise.

Reply:
agreed